apps/prairielearn/src/pages/instructorAssessmentInstance/instructorAssessmentInstance.sql

```diff
@@ -25,13 +25,6 @@ FROM
   JOIN course_instances AS ci ON (a.course_instance_id = ci.id)
   LEFT JOIN groups AS g ON (g.id = ai.group_id)
   LEFT JOIN group_users AS gu ON (g.id = gu.group_id)
-  JOIN enrollments AS e ON (
-    (
-      (ai.user_id = e.user_id)
-      OR (e.user_id = gu.user_id)
-    )
-    AND ci.id = e.course_instance_id
-  )
```

Comment:
This was a bug: if the user associated with this instance was not enrolled, the stats would be empty, but there's no reason to do that.

Reply:
I checked the blame history, this is to ensure stats only include assessment instances associated to enrolled students, not staff or dropped students.

Reply:
These are the stats for all instance questions in a single specific assessment instance, not across all instances.

Reply:
Ah, right, that makes sense. It's just that I saw that when this was introduced, `e` was retrieved to get the user role, which in this case does not apply. That said, if you remove it, I don't think you need the group or group user joins either, right? Or the check on soft-delete for group?

Reply:
Very good call! Removed those extra joins in 2907ca4.